docs/iris/src/whatsnew/1.11.rst

```diff
@@ -0,0 +1,34 @@
+What's New in Iris 1.11
+***********************
+
+:Release: 1.11
+:Date: 2016-11-28
+
+This document explains the new/changed features of Iris in version 1.11
+(:doc:`View all changes <index>`.)
+
+Iris 1.11 Features
+==================
+* If available, display the ``STASH`` code instead of ``unknown / (unknown)`` when printing cubes
+  with no ``standard_name`` and no ``units``.
+* Support for saving to netCDF with data packing has been added.
+* The coordinate system :class:`iris.coord_systems.LambertAzimuthalEqualArea` has been added with NetCDF saving support.
+
+Bugs Fixed
+==========
+* Fixed a floating point tolerance bug in :func:`iris.experimental.regrid.regrid_area_weighted_rectilinear_src_and_grid`
+  for wrapped longitudes.
+* Allow :func:`iris.util.new_axis` to promote the nominated scalar coordinate of a cube
+  with a scalar masked constant data payload.
+* Fixed a bug where :func:`iris.util._is_circular` would erroneously return false
+  when coordinate values are decreasing.
+* When saving to NetCDF, the existing behaviour of writing string attributes as ASCII has been
```

Comment:
This is a very long and quite unusual whatsnew entry. It takes a lot of time to say very little at all, and I'm really not a fan of the unprecedented PR link in the entry. Options...

 * leave it as it is
 * remove it entirely
 * only keep the first sentence of the entry.

I favour the last of those options - it says about the same amount of very little in a single sentence, which at least means we are acknowledging that a change has been made.